src/Tribe/Assets.php

```diff
@@ -78,13 +81,29 @@ public function enqueue() {
 				continue;
 			}
 
+			// If this asset was late called
+			// if ( ! $asset->is_registered ) {
+			// 	$this->register_in_wp( $asset );
+			// }
+
+			// By default we will enqueue this asset because it's already
```

Comment:
Not sure if I'm overthinking this, but reading through with fresh eyes I wonder if it would be clearer to make this comment something like:

```
// Default to enqueuing the asset if there are no conditionals,
// and default to not enqueuing it if there *are* conditionals
```